client/api/api.go

```diff
@@ -43,21 +41,23 @@ type errorReply struct {
 // CheckResponse returns an error (of type *Error) if the response
 // status code is not 2xx.
 func CheckResponse(res *http.Response) error {
-	if res.StatusCode >= 200 && res.StatusCode <= 299 {
+	const MinValidStatusCode = 200
```

Comment:
why define a const here?

Reply:
why make them pubic?

Reply:
Those are inside CheckResponse so they are not public. The same values are used when testing against this function so it's easier to maintain and read to define them as const.

Reply:
You are right, they are not public. However, they are not used anywhere else, are they?

Reply:
in this case, why make the change here then?

Reply:
It's easier to read to have a defined const instead of a raw value to me. It's personal preference I don't have strong opinion about it.

Reply:
Synced with Ryan offline and this doesn't seems to be an issue so I would resolve this conversation.